[2021-08-12 10:35:07,877] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-01-11T00:00:00+00:00 [queued]>
[2021-08-12 10:35:07,881] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-01-11T00:00:00+00:00 [queued]>
[2021-08-12 10:35:07,881] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:35:07,881] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 10:35:07,881] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:35:07,888] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_2> on 2021-01-11T00:00:00+00:00
[2021-08-12 10:35:07,890] {standard_task_runner.py:52} INFO - Started process 130702 to run task
[2021-08-12 10:35:07,893] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', '2021-01-11T00:00:00+00:00', '--job-id', '60', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/subdag_example.py', '--cfg-path', '/tmp/tmpqigxswjf', '--error-file', '/tmp/tmp9630zh54']
[2021-08-12 10:35:07,894] {standard_task_runner.py:77} INFO - Job 60: Subtask task_2
[2021-08-12 10:35:07,913] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_2 2021-01-11T00:00:00+00:00 [running]> on host adam
[2021-08-12 10:35:07,942] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-01-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-11T00:00:00+00:00
[2021-08-12 10:35:07,944] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 10:35:07,944] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo task 2']
[2021-08-12 10:35:07,952] {subprocess.py:74} INFO - Output:
[2021-08-12 10:35:07,953] {subprocess.py:78} INFO - task 2
[2021-08-12 10:35:07,954] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 10:35:07,969] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20210111T000000, start_date=20210812T083507, end_date=20210812T083507
[2021-08-12 10:35:07,984] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-12 10:35:08,024] {local_task_job.py:149} INFO - Task exited with return code 0
